templates/chrony.conf.j2

```diff
@@ -1,8 +1,13 @@
 # {{ ansible_managed }}
 
 # List of NTP servers to use.
-{% for server in chrony_config_server %}
-server {{ server }} iburst
+{% for item in chrony_config_server %}
+{%   if item['options'] is not defined %}
+server {{ item['server'] }}
+{%   elif item['options'] is defined %}
+server {{ item['server'] }}{% for opt in item['options'] %} {{ opt['option'] }}{% if opt['val'] is defined %} {{ opt['val'] }}{% endif %}{% endfor %}
+
+{%   endif %}
 {% endfor %}
```

Comment:
maybe you can do it like:
```suggestion
{% for item in chrony_config_server %}
server {{ item['server'] }}
{% if item['options'] is defined %}
{% for opt in item['options'] %} {{ opt['option'] }}{% if opt['val'] is defined %} {{ opt['val'] }}{% endif %}{% endfor %}
{% endif %}
{% endfor %}
```

Reply:
Given values:
````chrony_config_server:
  - server: ntpserver1
    options: 
    - option: "iburst"
       val: ""
    - option: "minpoll"
       val: "4"
    - option: maxpol
       val: "7"
- server: ntpserver2
    options: 
    - option: "iburst"
       val: ""
    - option: "minpoll"
       val: "4"
    - option: maxpol
       val: "7"
- server: ntpserver3
    options: 
    - option: "iburst"
       val: ""
    - option: "minpoll"
       val: "4"
    - option: maxpol
       val: "7"
````

That code renders chrony.conf settings:
````
    server ntpserver1
             iburst  minpoll 4 maxpoll 7        server 89.21.22.235
             iburst  minpoll 4 maxpoll 7        server 89.21.22.234
             iburst  minpoll 4 maxpoll 7`
````

and Chrony service fails to start ("control process exited with error code").  Might be possible to fix the line spacing, but not sure it's worth the effort for one line less code (one line is longer and more complex too)

Reply:
ok, the line-spacing seems to work with this:
````{% for item in chrony_config_server %}

    server {{ item['server'] }}  {% if item['options'] is defined -%}   {% for opt in item['options'] %} {{ opt['option'] }}{% if opt['val'] is defined %} {{ opt['val'] }}{% endif %}{% endfor %}
    {%- endif %}
{% endfor %}````

But one line is a bit long.  Would you prefer this, or as originally committed?

Reply:
the last one LGTM